tests/Database/Base.php

```
@@ -1845,7 +1846,7 @@ public function testFindContains(): void
          * Array contains OR condition
          */
         $documents = static::getDatabase()->find('movies', [
-            Query::contains('generes', ['comics', 'kids']),
+            Query::contains('genres', ['comics', 'kids']),
```

Will Query::contains on nonarray attribute should throw an Exception?

---

Not sure about this one 🤔 I think it should, the other option could be treating it as `LIKE` on a non-array attribute?

---

I think we should add a test for :)

---

I tested a query on 
$documents = static::getDatabase()->find('movies', [
                Query::contains('name', ['captain']),
            ]);
            No errors but no results

---

Fixed :+1: